serverless/knative_eventing/serverless-subscriptions.adoc

```diff
@@ -6,6 +6,57 @@ include::modules/common-attributes.adoc[]
 
 toc::[]
 
-Subscriptions deliver events to event sinks from a Channel.
+You can create a subscription to enable event delivery from a channel to a service or other event sink.
 
-include::modules/serverless-creating-subscriptions.adoc[leveloffset=+1]
+[IMPORTANT]
+====
+Knative Eventing is a Technology Preview feature. The InMemoryChannel type is provided for development use only, and should not be used in a production environment.
+====
+
+.Prerequisites
+
+* You must have a current installation of xref:../../serverless/getting_started/installing-openshift-serverless.adoc#serverless-install-web-console_installing-openshift-serverless[{ServerlessProductName}], including Knative Serving and Eventing, in your {product-title} cluster. This can be installed by a cluster administrator.
+* If you do not have an existing sink that you wish to use, create a Service to use as a sink by following the documentation on xref:../../serverless/getting_started/serving-creating-managing-apps.adoc#serving-creating-managing-apps[Creating and managing serverless applications].
+* You must have a channel to connect your subscription to. See xref:../../serverless/knative_eventing/serverless-channels.adoc#serverless-channels[Using channels with Knative Eventing].
+
+.Procedure
+
+. Create a Subscription object to connect a channel to a service, by creating a YAML file containing the following:
++
+[source,yaml]
+----
+apiVersion: messaging.knative.dev/v1beta1
+kind: Subscription
+metadata:
+  name: my-subscription <1>
+  namespace: default
+spec:
+  channel: <2>
+    apiVersion: messaging.knative.dev/v1beta1
+    kind: Channel
+    name: example-channel
+  delivery: <3>
+    deadLetterSink:
+      ref:
+        apiVersion: serving.knative.dev/v1
+        kind: Service
+        name: error-handler
+  subscriber: <4>
+    ref:
+      apiVersion: serving.knative.dev/v1
+      kind: Service
+      name: event-display
+----
++
+<1> Name of the subscription.
+<2> Configuration settings for the channel that the subscription connects to.
+<3> Configuration settings for event delivery. This tells the subscription what happens to events that cannot be delivered to the subscriber. When this is configured, events that failed to be consumed are sent to the `deadLetterSink`. The event is dropped, no re-delivery of the event is attempted, and an error is logged in the system. The `deadLetterSink` value must be a link:https://pkg.go.dev/knative.dev/pkg/apis/duck/v1?tab=doc#Destination[Destination].
```

Comment:
Not sure if this would need an ack from Product Experience/Support because it is an external link?

Reply:
Removed for now.